### DISCUSSION TITLE
Logitech Media Server as a music source

### DISCUSSION THREAD

**Discussion by asjmcguire** [music-assistant/support#406] [Feature requests and ideas]
I know the eventual aim of this project is to replace the need to have a Logitech Media Server, but until it gains more feature parity, and important things like synchronised playback and transfer queue to a different player - it would be nice to be able to keep my Logitech Media Sever and have Music Assistant treat it as a music provider - since it does have a public API and provides HTTP access to the actual music files.

Thanks for considering this.

**Comment by marcelveldt** [comment]
Tbh I'd rather invest my time in implementing the missing features then let MA connect to all existing similar products out there.
I think that if LMS currently suits your needs you should stick with that for now. Offcourse I'm open if somebody want to implement the music provider plugin for MA. It's just that I will not likely invest my time in that.
